src/backend/vulkan/src/lib.rs

```diff
@@ -61,7 +61,7 @@ lazy_static! {
             CStr::from_bytes_with_nul(b"VK_LAYER_GOOGLE_unique_objects\0").unwrap(),
         ]
     } else if cfg!(debug_assertions) {
-        vec![CStr::from_bytes_with_nul(b"VK_LAYER_LUNARG_standard_validation\0").unwrap()]
+        vec![CStr::from_bytes_with_nul(b"VK_LAYER_KHRONOS_validation\0").unwrap()]
```

Comment:
the layer list above also needs to be update with regards to the deprecated extensions, could we have it here?

Reply:
Huh, completely missed that, sorry! I will remove the upper if branch.

Reply:
would you mind squashing the commits?